tox.ini

```diff
@@ -56,7 +56,7 @@ commands =
     behave-lxd-18.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.bionic,series.lts,series.all" --tags="~upgrade"
     behave-lxd-20.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.focal,series.lts,series.all" --tags="~upgrade"
     behave-lxd-21.10: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.impish,series.all" --tags="~upgrade"
-    behave-lxd-22.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.jammy,series.all" --tags="~upgrade"
+    behave-lxd-22.04: behave -v {posargs} --tags="uses.config.machine_type.lxd.container" --tags="series.jammy,series.lts,series.all" --tags="~upgrade"
```

Comment:
Should we wait a bit to add the LTS tag here ? We do have some tests on LTS releases were we enable some services and will not work on jammy at this point

Reply:
It's ok since we don't specify Jammy in the `releases` examples for the Scenario Outlines - when we have those tests for Jammy, it is a matter of listing it there

Reply:
Fair enough, good point